pom.xml

```diff
@@ -22,159 +24,190 @@
-            <artifactId>jackson-core</artifactId>
-            <version>2.9.6</version>
+            <groupId>uk.gov.ons.ctp.product</groupId>
+            <artifactId>partysvc-api</artifactId>
+            <version>10.50.3</version>
         </dependency>
 
         <dependency>
-            <groupId>org.springframework.integration</groupId>
-            <artifactId>spring-integration-core</artifactId>
+            <groupId>uk.gov.ons.ctp.product</groupId>
+            <artifactId>actionsvc-api</artifactId>
+            <version>10.49.20</version>
         </dependency>
 
         <dependency>
-            <groupId>org.springframework.integration</groupId>
-            <artifactId>spring-integration-xml</artifactId>
+            <groupId>uk.gov.ons.ctp.product</groupId>
+            <artifactId>casesvc-api</artifactId>
+            <version>10.49.25</version>
         </dependency>
 
         <dependency>
-            <groupId>org.springframework.integration</groupId>
-            <artifactId>spring-integration-file</artifactId>
+            <groupId>uk.gov.ons.ctp.product</groupId>
+            <artifactId>collectionexercisesvc-api</artifactId>
+            <version>10.49.7</version>
         </dependency>
 
         <dependency>
-            <groupId>org.springframework.integration</groupId>
-            <artifactId>spring-integration-http</artifactId>
+            <groupId>uk.gov.ons.ctp.product</groupId>
+            <artifactId>surveysvc-api</artifactId>
+            <version>10.49.2</version>
         </dependency>
 
         <dependency>
-            <groupId>org.springframework.integration</groupId>
-            <artifactId>spring-integration-stream</artifactId>
+            <groupId>uk.gov.ons.ctp.product</groupId>
+            <artifactId>reportmodule</artifactId>
+            <version>10.49.0</version>
         </dependency>
 
         <dependency>
-            <groupId>org.springframework.integration</groupId>
-            <artifactId>spring-integration-amqp</artifactId>
+            <groupId>uk.gov.ons.ctp.product</groupId>
+            <artifactId>samplesvc-api</artifactId>
+            <version>10.49.16</version>
         </dependency>
 
         <dependency>
-            <groupId>org.springframework.cloud</groupId>
-            <artifactId>spring-cloud-spring-service-connector</artifactId>
+            <groupId>uk.gov.ons.ctp.common</groupId>
+            <artifactId>framework</artifactId>
+            <version>10.49.14</version>
         </dependency>
 
-        <!-- If you intend to deploy the app on Cloud Foundry, add the following -->
         <dependency>
-            <groupId>org.springframework.cloud</groupId>
-            <artifactId>spring-cloud-cloudfoundry-connector</artifactId>
+            <groupId>uk.gov.ons.tools</groupId>
+            <artifactId>rabbit</artifactId>
+            <version>1.0.1</version>
         </dependency>
 
+        <!-- ONS END -->
+
+        <!-- third party libraries -->
```

Comment:
Any reason this can't be Third party libraries to match the closing comment?

Reply:
copy-paste is why  ;o)